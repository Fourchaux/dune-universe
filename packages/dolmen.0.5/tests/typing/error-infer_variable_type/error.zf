# Rejected by Dolmen because it doesn't do inference for the
# types of bound variables, even if it could be possible

val t : type.
val p : t -> prop.

assert forall a. p a.

