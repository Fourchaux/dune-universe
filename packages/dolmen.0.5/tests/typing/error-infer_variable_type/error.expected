File "tests/typing/error-infer_variable_type/error.zf", line 7, character 14-15:
Error Cannot infer type for a quantified variable
